sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/AbstractRulesDefinition.java

```diff
@@ -106,9 +110,37 @@ private void addSecurityStandards(NewRule rule, SecurityStandards securityStanda
         rule.addOwaspTop10(RulesDefinition.OwaspTop10Version.Y2021, RulesDefinition.OwaspTop10.valueOf(s));
       }
     }
+
+    addPciDss(rule, securityStandards);
+    addASVS(rule, securityStandards);
```

Comment:
The program structure is now irregular. Three standards are added directly in this method, two more are delegated to methods. 
I'd be fine with adding them all here with positive condition logic

Reply:
..and get rid of them eventually

Reply:
I've extracted all the standards imports to methods.